.drone.yml

```
@@ -451,3 +451,11 @@ services:
     image: gitea/test-openldap:latest
     when:
       event: [ push, tag, pull_request ]
+
+  elasticsearch:
+    image: elastic/elasticsearch:5.6.16
+    environment:
+      # Disable production checks because they are not relevant and fail
+      ES_JAVA_OPTS: "-Des.enforce.bootstrap.checks=true"
```

My fault.. should have tested it locally.

This is the official way to start ES in development mode (disabling bootstrap checks) 

```suggestion
      - "discovery.type=single-node"
```